.gitignore

```diff
@@ -16,6 +16,7 @@ eggs/
 .eggs/
 lib/
 lib64/
+src/**/migrations/
```

Comment:
Not including migrations in the repository is actually a bad practice. This becomes an issue when you need to for example have a migration that does data manipulation (e.g. https://docs.djangoproject.com/en/3.2/howto/writing-migrations/#migrations-that-add-unique-fields)
If not everyone has the same set of migrations, it becomes a huge hassle to do it.
If we are running this in staging/prod we should have a consistent migration history and the way we achieve that is by committing the migration files.
Django itself as well as every third-party module I am aware of ship with their migration files pre-generated for this reason.

Reply:
Removed this. Thanks!